genders/fr-FR.json

```diff
@@ -0,0 +1,44 @@
+{
+  "?": "?",
+  "agender": "agenre",
+  "androgyne": "androgyne",
+  "androgynous": "androgyne",
+  "bigender": "bigenre",
+  "cis": "cis",
+  "cisgender": "cisgenre",
+  "decline to provide": "ne souhaite pas le fournir",
+  "f to m": "f to m",
+  "female": "femme",
+  "female to male": "female to male",
```

Comment:
should this be included in a French translation with the English words, or omitted entirely?